Neos.Neos/Classes/Fusion/Helper/NodeHelper.php

```diff
@@ -75,6 +76,17 @@ public function path(Node $node): string
         return AbsoluteNodePath::fromLeafNodeAndAncestors($node, $ancestors)->serializeToString();
     }
 
+    public function aggregateIdPath(Node $node): string
```

Comment:
there should be a comment _why_ this use might be necessary over `path`. And `path` should also scream in your face that it throws if there is no node name :D

Reply:
Done